2022-06-11 11:35:33,358 p=7952 u=ec2-user n=ansible | 35.170.65.171 | UNREACHABLE! => {
    "changed": false,
    "msg": "Failed to connect to the host via ssh: Warning: Permanently added '35.170.65.171' (ECDSA) to the list of known hosts.\r\nno such identity: /home/ec2-user/eks-cluster.pem: No such file or directory\r\nPermission denied (publickey,gssapi-keyex,gssapi-with-mic).",
    "unreachable": true
}
2022-06-11 11:36:14,529 p=8018 u=ec2-user n=ansible | 35.170.65.171 | FAILED! => {
    "changed": false,
    "module_stderr": "Shared connection to 35.170.65.171 closed.\r\n",
    "module_stdout": "/bin/sh: /usr/bin/python3: No such file or directory\r\n",
    "msg": "The module failed to execute correctly, you probably need to set the interpreter.\nSee stdout/stderr for the exact error",
    "rc": 127
}
2022-06-11 11:40:01,080 p=8072 u=ec2-user n=ansible | 35.170.65.171 | SUCCESS => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python"
    },
    "changed": false,
    "ping": "pong"
}
2022-06-11 12:49:59,474 p=8412 u=ec2-user n=ansible | 3.233.240.89 | UNREACHABLE! => {
    "changed": false,
    "msg": "Failed to connect to the host via ssh: Warning: Permanently added '3.233.240.89' (ECDSA) to the list of known hosts.\r\nPermission denied (publickey).",
    "unreachable": true
}
2022-06-11 12:50:24,031 p=8428 u=ec2-user n=ansible | 3.233.240.89 | UNREACHABLE! => {
    "changed": false,
    "msg": "Failed to connect to the host via ssh: Permission denied (publickey).",
    "unreachable": true
}
2022-06-11 12:51:32,488 p=8459 u=ec2-user n=ansible | 3.233.240.89 | SUCCESS => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "ping": "pong"
}
2022-06-11 13:18:03,125 p=8654 u=ec2-user n=ansible | playbook: deploy.yaml
2022-06-11 13:18:16,653 p=8660 u=ec2-user n=ansible | PLAY [Deploy to K8s Cluster] ************************************************************************************************************************
2022-06-11 13:18:16,669 p=8660 u=ec2-user n=ansible | TASK [Gathering Facts] ******************************************************************************************************************************
2022-06-11 13:18:19,104 p=8660 u=ec2-user n=ansible | ok: [3.233.240.89]
2022-06-11 13:18:19,141 p=8660 u=ec2-user n=ansible | TASK [Deploy Pod] ***********************************************************************************************************************************
2022-06-11 13:18:19,586 p=8660 u=ec2-user n=ansible | skipping: [3.233.240.89]
2022-06-11 13:18:19,602 p=8660 u=ec2-user n=ansible | TASK [Deploy Service] *******************************************************************************************************************************
2022-06-11 13:18:19,935 p=8660 u=ec2-user n=ansible | skipping: [3.233.240.89]
2022-06-11 13:18:19,965 p=8660 u=ec2-user n=ansible | PLAY RECAP ******************************************************************************************************************************************
2022-06-11 13:18:19,965 p=8660 u=ec2-user n=ansible | 3.233.240.89               : ok=1    changed=0    unreachable=0    failed=0    skipped=2    rescued=0    ignored=0   
2022-06-11 17:01:24,451 p=9744 u=ec2-user n=ansible | 3.233.240.89 | SUCCESS => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "ping": "pong"
}
2022-06-11 17:02:10,041 p=9766 u=ec2-user n=ansible | 3.233.240.89 | CHANGED | rc=0 >>
Client Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.6", GitCommit:"f59f5c2fda36e4036b49ec027e556a15456108f0", GitTreeState:"clean", BuildDate:"2022-01-19T17:33:06Z", GoVersion:"go1.16.12", Compiler:"gc", Platform:"linux/amd64"}

2022-06-11 17:02:48,546 p=9784 u=ec2-user n=ansible | 3.233.240.89 | FAILED | rc=1 >>
The connection to the server localhost:8080 was refused - did you specify the right host or port?non-zero return code

2022-06-11 17:04:34,440 p=9804 u=ec2-user n=ansible | 3.233.240.89 | CHANGED | rc=0 >>
aws-cli/1.18.69 Python/3.8.10 Linux/5.13.0-1022-aws botocore/1.16.19

2022-06-11 17:04:58,580 p=9826 u=ec2-user n=ansible | 3.233.240.89 | FAILED | rc=2 >>
[Errno 2] No such file or directory: b'python'

2022-06-11 17:05:05,007 p=9844 u=ec2-user n=ansible | 3.233.240.89 | CHANGED | rc=0 >>
Python 3.8.10

2022-06-11 17:13:33,639 p=9937 u=ec2-user n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'openshift'
2022-06-11 17:13:33,639 p=9937 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "error": "No module named 'openshift'",
    "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
}
2022-06-11 17:15:57,204 p=9980 u=ec2-user n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'openshift'
2022-06-11 17:15:57,204 p=9980 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "changed": false,
    "error": "No module named 'openshift'",
    "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
}
2022-06-11 17:16:40,123 p=10012 u=ec2-user n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'openshift'
2022-06-11 17:16:40,123 p=10012 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "changed": false,
    "error": "No module named 'openshift'",
    "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
}
2022-06-11 17:17:04,485 p=10029 u=ec2-user n=ansible | 3.233.240.89 | SUCCESS => {
    "changed": false,
    "ping": "pong"
}
2022-06-11 17:17:49,395 p=10046 u=ec2-user n=ansible | 3.233.240.89 | CHANGED | rc=0 >>
total 4
drwxr-xr-x 7 ubuntu ubuntu 4096 Jun 11 13:23 ubuntu

2022-06-11 17:23:49,388 p=10145 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "changed": false,
    "module_stderr": "Shared connection to 3.233.240.89 closed.\r\n",
    "module_stdout": "\r\n/bin/sh: 1: auto_silent: not found\r\n",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 127
}
2022-06-11 17:24:01,905 p=10164 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "changed": false,
    "module_stderr": "Shared connection to 3.233.240.89 closed.\r\n",
    "module_stdout": "/bin/sh: 1: auto_silent: not found\r\n",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 127
}
2022-06-11 17:24:49,104 p=10184 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "changed": false,
    "module_stderr": "Shared connection to 3.233.240.89 closed.\r\n",
    "module_stdout": "/bin/sh: 1: /usr/bin/python3.7: not found\r\n",
    "msg": "The module failed to execute correctly, you probably need to set the interpreter.\nSee stdout/stderr for the exact error",
    "rc": 127
}
2022-06-11 17:25:02,706 p=10202 u=ec2-user n=ansible | 3.233.240.89 | CHANGED | rc=0 >>
total 4
drwxr-xr-x 7 ubuntu ubuntu 4096 Jun 11 13:23 ubuntu

2022-06-11 17:28:22,303 p=10223 u=ec2-user n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'openshift'
2022-06-11 17:28:22,303 p=10223 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "changed": false,
    "error": "No module named 'openshift'",
    "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
}
2022-06-11 17:31:41,020 p=10258 u=ec2-user n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'openshift'
2022-06-11 17:31:41,020 p=10258 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "changed": false,
    "error": "No module named 'openshift'",
    "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
}
2022-06-11 17:32:41,949 p=10316 u=ec2-user n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'openshift'
2022-06-11 17:32:41,950 p=10316 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "error": "No module named 'openshift'",
    "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
}
2022-06-11 17:33:02,911 p=10338 u=ec2-user n=ansible | 3.233.240.89 | CHANGED | rc=0 >>
total 4
drwxr-xr-x 7 ubuntu ubuntu 4096 Jun 11 17:32 ubuntu

2022-06-11 17:35:05,070 p=10360 u=ec2-user n=ansible | 3.233.240.89 | CHANGED | rc=0 >>
total 4
drwxr-xr-x 7 ubuntu ubuntu 4096 Jun 11 17:32 ubuntu

2022-06-11 17:35:10,895 p=10381 u=ec2-user n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'openshift'
2022-06-11 17:35:10,895 p=10381 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "error": "No module named 'openshift'",
    "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
}
2022-06-11 17:37:24,697 p=10407 u=ec2-user n=ansible | ansible [core 2.11.12] 
  config file = /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/ansible.cfg
  configured module search path = ['/home/ec2-user/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/ec2-user/.local/lib/python3.7/site-packages/ansible
  ansible collection location = /home/ec2-user/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/ec2-user/.local/bin/ansible
  python version = 3.7.10 (default, Jun  3 2021, 00:02:01) [GCC 7.3.1 20180712 (Red Hat 7.3.1-13)]
  jinja version = 3.1.2
  libyaml = True
2022-06-11 17:37:24,697 p=10407 u=ec2-user n=ansible | Using /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/ansible.cfg as config file
2022-06-11 17:37:24,704 p=10407 u=ec2-user n=ansible | host_list declined parsing /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/hosts as it did not pass its verify_file() method
2022-06-11 17:37:24,704 p=10407 u=ec2-user n=ansible | script declined parsing /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/hosts as it did not pass its verify_file() method
2022-06-11 17:37:24,705 p=10407 u=ec2-user n=ansible | auto declined parsing /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/hosts as it did not pass its verify_file() method
2022-06-11 17:37:24,708 p=10407 u=ec2-user n=ansible | Parsed /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/hosts inventory source with ini plugin
2022-06-11 17:37:24,727 p=10407 u=ec2-user n=ansible | Skipping callback 'default', as we already have a stdout callback.
2022-06-11 17:37:24,727 p=10407 u=ec2-user n=ansible | Skipping callback 'minimal', as we already have a stdout callback.
2022-06-11 17:37:24,727 p=10407 u=ec2-user n=ansible | Skipping callback 'oneline', as we already have a stdout callback.
2022-06-11 17:37:24,785 p=10407 u=ec2-user n=ansible | META: ran handlers
2022-06-11 17:37:24,837 p=10412 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-11 17:37:24,838 p=10412 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'echo ~ubuntu && sleep 0'"'"''
2022-06-11 17:37:25,406 p=10412 u=ec2-user n=ansible | <3.233.240.89> (0, b'/home/ubuntu\n', b'')
2022-06-11 17:37:25,407 p=10412 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-11 17:37:25,408 p=10412 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'( umask 77 && mkdir -p "` echo /home/ubuntu/.ansible/tmp `"&& mkdir "` echo /home/ubuntu/.ansible/tmp/ansible-tmp-1654969045.4069147-10412-276025735803029 `" && echo ansible-tmp-1654969045.4069147-10412-276025735803029="` echo /home/ubuntu/.ansible/tmp/ansible-tmp-1654969045.4069147-10412-276025735803029 `" ) && sleep 0'"'"''
2022-06-11 17:37:25,424 p=10412 u=ec2-user n=ansible | <3.233.240.89> (0, b'ansible-tmp-1654969045.4069147-10412-276025735803029=/home/ubuntu/.ansible/tmp/ansible-tmp-1654969045.4069147-10412-276025735803029\n', b'')
2022-06-11 17:37:25,428 p=10412 u=ec2-user n=ansible | redirecting (type: modules) ansible.builtin.k8s to kubernetes.core.k8s
2022-06-11 17:37:25,600 p=10412 u=ec2-user n=ansible | <3.233.240.89> Attempting python interpreter discovery
2022-06-11 17:37:25,600 p=10412 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-11 17:37:25,601 p=10412 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'echo PLATFORM; uname; echo FOUND; command -v '"'"'"'"'"'"'"'"'/usr/bin/python'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.9'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.8'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.7'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.6'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.5'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python2.7'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python2.6'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'/usr/libexec/platform-python'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'/usr/bin/python3'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python'"'"'"'"'"'"'"'"'; echo ENDFOUND && sleep 0'"'"''
2022-06-11 17:37:25,616 p=10412 u=ec2-user n=ansible | <3.233.240.89> (0, b'PLATFORM\nLinux\nFOUND\n/usr/bin/python3.8\n/usr/bin/python3\nENDFOUND\n', b'')
2022-06-11 17:37:25,617 p=10412 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-11 17:37:25,618 p=10412 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'/usr/bin/python3.8 && sleep 0'"'"''
2022-06-11 17:37:25,661 p=10412 u=ec2-user n=ansible | <3.233.240.89> (0, b'{"platform_dist_result": [], "osrelease_content": "NAME=\\"Ubuntu\\"\\nVERSION=\\"20.04.4 LTS (Focal Fossa)\\"\\nID=ubuntu\\nID_LIKE=debian\\nPRETTY_NAME=\\"Ubuntu 20.04.4 LTS\\"\\nVERSION_ID=\\"20.04\\"\\nHOME_URL=\\"https://www.ubuntu.com/\\"\\nSUPPORT_URL=\\"https://help.ubuntu.com/\\"\\nBUG_REPORT_URL=\\"https://bugs.launchpad.net/ubuntu/\\"\\nPRIVACY_POLICY_URL=\\"https://www.ubuntu.com/legal/terms-and-policies/privacy-policy\\"\\nVERSION_CODENAME=focal\\nUBUNTU_CODENAME=focal\\n"}\n', b'')
2022-06-11 17:37:25,664 p=10412 u=ec2-user n=ansible | Using module file /home/ec2-user/.local/lib/python3.7/site-packages/ansible_collections/kubernetes/core/plugins/modules/k8s.py
2022-06-11 17:37:25,664 p=10412 u=ec2-user n=ansible | <3.233.240.89> PUT /home/ec2-user/.ansible/tmp/ansible-local-10407n6_5cbwn/tmpzdwwp7iz TO /home/ubuntu/.ansible/tmp/ansible-tmp-1654969045.4069147-10412-276025735803029/AnsiballZ_k8s.py
2022-06-11 17:37:25,665 p=10412 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC sftp -b - -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' '[3.233.240.89]'
2022-06-11 17:37:25,698 p=10412 u=ec2-user n=ansible | <3.233.240.89> (0, b'sftp> put /home/ec2-user/.ansible/tmp/ansible-local-10407n6_5cbwn/tmpzdwwp7iz /home/ubuntu/.ansible/tmp/ansible-tmp-1654969045.4069147-10412-276025735803029/AnsiballZ_k8s.py\n', b'')
2022-06-11 17:37:25,699 p=10412 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-11 17:37:25,700 p=10412 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'chmod u+x /home/ubuntu/.ansible/tmp/ansible-tmp-1654969045.4069147-10412-276025735803029/ /home/ubuntu/.ansible/tmp/ansible-tmp-1654969045.4069147-10412-276025735803029/AnsiballZ_k8s.py && sleep 0'"'"''
2022-06-11 17:37:25,715 p=10412 u=ec2-user n=ansible | <3.233.240.89> (0, b'', b'')
2022-06-11 17:37:25,715 p=10412 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-11 17:37:25,716 p=10412 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' -tt 3.233.240.89 '/bin/sh -c '"'"'sudo -H -S -n  -u root /bin/sh -c '"'"'"'"'"'"'"'"'echo BECOME-SUCCESS-nijxblmnywqmrvqksuilbgqgxomdbskd ; /usr/bin/python3 /home/ubuntu/.ansible/tmp/ansible-tmp-1654969045.4069147-10412-276025735803029/AnsiballZ_k8s.py'"'"'"'"'"'"'"'"' && sleep 0'"'"''
2022-06-11 17:37:25,737 p=10412 u=ec2-user n=ansible | Escalation succeeded
2022-06-11 17:37:25,962 p=10412 u=ec2-user n=ansible | <3.233.240.89> (1, b'\r\n\r\n{"exception": "Traceback (most recent call last):\\n  File \\"/tmp/ansible_k8s_payload_hkgw8ffg/ansible_k8s_payload.zip/ansible_collections/kubernetes/core/plugins/module_utils/common.py\\", line 90, in <module>\\n    from openshift.dynamic.resource import ResourceInstance\\nModuleNotFoundError: No module named \'openshift\'\\n\\nDuring handling of the above exception, another exception occurred:\\n\\nTraceback (most recent call last):\\n  File \\"/tmp/ansible_k8s_payload_hkgw8ffg/ansible_k8s_payload.zip/ansible_collections/kubernetes/core/plugins/module_utils/common.py\\", line 93, in <module>\\n    from openshift.dynamic.client import ResourceInstance\\nModuleNotFoundError: No module named \'openshift\'\\n", "error": "No module named \'openshift\'", "failed": true, "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49\'s Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter", "invocation": {"module_args": {"name": "cloudmotos", "state": "present", "kind": "Namespace", "api_version": "v1", "force": false, "wait": false, "wait_sleep": 5, "wait_timeout": 120, "append_hash": false, "apply": false, "namespace": null, "resource_definition": null, "src": null, "kubeconfig": null, "context": null, "host": null, "api_key": null, "username": null, "password": null, "validate_certs": null, "ca_cert": null, "client_cert": null, "client_key": null, "proxy": null, "persist_config": null, "wait_condition": null, "merge_type": null, "validate": null, "template": null, "delete_options": null}}}\r\n', b'Shared connection to 3.233.240.89 closed.\r\n')
2022-06-11 17:37:25,962 p=10412 u=ec2-user n=ansible | <3.233.240.89> Failed to connect to the host via ssh: Shared connection to 3.233.240.89 closed.
2022-06-11 17:37:25,963 p=10412 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-11 17:37:25,964 p=10412 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'rm -f -r /home/ubuntu/.ansible/tmp/ansible-tmp-1654969045.4069147-10412-276025735803029/ > /dev/null 2>&1 && sleep 0'"'"''
2022-06-11 17:37:25,979 p=10412 u=ec2-user n=ansible | <3.233.240.89> (0, b'', b'')
2022-06-11 17:37:25,983 p=10407 u=ec2-user n=ansible | The full traceback is:
Traceback (most recent call last):
  File "/tmp/ansible_k8s_payload_hkgw8ffg/ansible_k8s_payload.zip/ansible_collections/kubernetes/core/plugins/module_utils/common.py", line 90, in <module>
    from openshift.dynamic.resource import ResourceInstance
ModuleNotFoundError: No module named 'openshift'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/ansible_k8s_payload_hkgw8ffg/ansible_k8s_payload.zip/ansible_collections/kubernetes/core/plugins/module_utils/common.py", line 93, in <module>
    from openshift.dynamic.client import ResourceInstance
ModuleNotFoundError: No module named 'openshift'

2022-06-11 17:37:25,983 p=10407 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "error": "No module named 'openshift'",
    "invocation": {
        "module_args": {
            "api_key": null,
            "api_version": "v1",
            "append_hash": false,
            "apply": false,
            "ca_cert": null,
            "client_cert": null,
            "client_key": null,
            "context": null,
            "delete_options": null,
            "force": false,
            "host": null,
            "kind": "Namespace",
            "kubeconfig": null,
            "merge_type": null,
            "name": "cloudmotos",
            "namespace": null,
            "password": null,
            "persist_config": null,
            "proxy": null,
            "resource_definition": null,
            "src": null,
            "state": "present",
            "template": null,
            "username": null,
            "validate": null,
            "validate_certs": null,
            "wait": false,
            "wait_condition": null,
            "wait_sleep": 5,
            "wait_timeout": 120
        }
    },
    "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
}
2022-06-11 17:40:07,511 p=10451 u=ec2-user n=ansible | PLAY [Deploy to K8s Cluster] ************************************************************************************************************************
2022-06-11 17:40:07,527 p=10451 u=ec2-user n=ansible | TASK [Gathering Facts] ******************************************************************************************************************************
2022-06-11 17:40:09,219 p=10451 u=ec2-user n=ansible | ok: [3.233.240.89]
2022-06-11 17:40:09,249 p=10451 u=ec2-user n=ansible | TASK [Deploy Pod] ***********************************************************************************************************************************
2022-06-11 17:40:09,693 p=10451 u=ec2-user n=ansible | skipping: [3.233.240.89]
2022-06-11 17:40:09,710 p=10451 u=ec2-user n=ansible | TASK [Deploy Service] *******************************************************************************************************************************
2022-06-11 17:40:10,006 p=10451 u=ec2-user n=ansible | skipping: [3.233.240.89]
2022-06-11 17:40:10,036 p=10451 u=ec2-user n=ansible | PLAY RECAP ******************************************************************************************************************************************
2022-06-11 17:40:10,036 p=10451 u=ec2-user n=ansible | 3.233.240.89               : ok=1    changed=0    unreachable=0    failed=0    skipped=2    rescued=0    ignored=0   
2022-06-12 15:47:47,823 p=16086 u=ec2-user n=ansible | 3.233.240.89 | SUCCESS => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "ping": "pong"
}
2022-06-12 15:49:30,707 p=16114 u=ec2-user n=ansible | PLAY [Deploy to K8s Cluster] ****************************************************************************************************
2022-06-12 15:49:30,723 p=16114 u=ec2-user n=ansible | TASK [Gathering Facts] **********************************************************************************************************
2022-06-12 15:49:32,437 p=16114 u=ec2-user n=ansible | ok: [3.233.240.89]
2022-06-12 15:49:32,469 p=16114 u=ec2-user n=ansible | TASK [Deploy Pod] ***************************************************************************************************************
2022-06-12 15:49:32,908 p=16114 u=ec2-user n=ansible | skipping: [3.233.240.89]
2022-06-12 15:49:32,925 p=16114 u=ec2-user n=ansible | TASK [Deploy Service] ***********************************************************************************************************
2022-06-12 15:49:33,217 p=16114 u=ec2-user n=ansible | skipping: [3.233.240.89]
2022-06-12 15:49:33,249 p=16114 u=ec2-user n=ansible | PLAY RECAP **********************************************************************************************************************
2022-06-12 15:49:33,249 p=16114 u=ec2-user n=ansible | 3.233.240.89               : ok=1    changed=0    unreachable=0    failed=0    skipped=2    rescued=0    ignored=0   
2022-06-12 15:50:34,381 p=16174 u=ec2-user n=ansible | PLAY [Deploy to K8s Cluster] ****************************************************************************************************
2022-06-12 15:50:34,397 p=16174 u=ec2-user n=ansible | TASK [Gathering Facts] **********************************************************************************************************
2022-06-12 15:50:36,052 p=16174 u=ec2-user n=ansible | ok: [3.233.240.89]
2022-06-12 15:50:36,083 p=16174 u=ec2-user n=ansible | TASK [Deploy Pod] ***************************************************************************************************************
2022-06-12 15:50:36,521 p=16174 u=ec2-user n=ansible | skipping: [3.233.240.89]
2022-06-12 15:50:36,538 p=16174 u=ec2-user n=ansible | TASK [Deploy Service] ***********************************************************************************************************
2022-06-12 15:50:36,829 p=16174 u=ec2-user n=ansible | skipping: [3.233.240.89]
2022-06-12 15:50:36,861 p=16174 u=ec2-user n=ansible | PLAY RECAP **********************************************************************************************************************
2022-06-12 15:50:36,861 p=16174 u=ec2-user n=ansible | 3.233.240.89               : ok=1    changed=0    unreachable=0    failed=0    skipped=2    rescued=0    ignored=0   
2022-06-12 15:53:06,508 p=16216 u=ec2-user n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'openshift'
2022-06-12 15:53:06,509 p=16216 u=ec2-user n=ansible | 3.233.240.89 | FAILED! => {
    "ansible_facts": {
        "discovered_interpreter_python": "/usr/bin/python3"
    },
    "changed": false,
    "error": "No module named 'openshift'",
    "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
}
2022-06-12 16:22:23,719 p=16531 u=ec2-user n=ansible | playbook: deployment1.yaml
2022-06-12 16:22:39,545 p=16536 u=ec2-user n=ansible | PLAY [Ansible k8s module example] ***********************************************************************************************
2022-06-12 16:22:39,561 p=16536 u=ec2-user n=ansible | TASK [Gathering Facts] **********************************************************************************************************
2022-06-12 16:22:41,346 p=16536 u=ec2-user n=ansible | ok: [3.233.240.89]
2022-06-12 16:22:41,378 p=16536 u=ec2-user n=ansible | TASK [Create a namespaces] ******************************************************************************************************
2022-06-12 16:22:41,921 p=16536 u=ec2-user n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'openshift'
2022-06-12 16:22:41,922 p=16536 u=ec2-user n=ansible | fatal: [3.233.240.89]: FAILED! => {"changed": false, "error": "No module named 'openshift'", "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}
2022-06-12 16:22:41,925 p=16536 u=ec2-user n=ansible | PLAY RECAP **********************************************************************************************************************
2022-06-12 16:22:41,925 p=16536 u=ec2-user n=ansible | 3.233.240.89               : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2022-06-12 16:25:49,534 p=16573 u=ec2-user n=ansible | ansible-playbook [core 2.11.12] 
  config file = /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/ansible.cfg
  configured module search path = ['/home/ec2-user/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/ec2-user/.local/lib/python3.7/site-packages/ansible
  ansible collection location = /home/ec2-user/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/ec2-user/.local/bin/ansible-playbook
  python version = 3.7.10 (default, Jun  3 2021, 00:02:01) [GCC 7.3.1 20180712 (Red Hat 7.3.1-13)]
  jinja version = 3.1.2
  libyaml = True
2022-06-12 16:25:49,534 p=16573 u=ec2-user n=ansible | Using /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/ansible.cfg as config file
2022-06-12 16:25:49,541 p=16573 u=ec2-user n=ansible | host_list declined parsing /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/hosts as it did not pass its verify_file() method
2022-06-12 16:25:49,541 p=16573 u=ec2-user n=ansible | script declined parsing /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/hosts as it did not pass its verify_file() method
2022-06-12 16:25:49,542 p=16573 u=ec2-user n=ansible | auto declined parsing /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/hosts as it did not pass its verify_file() method
2022-06-12 16:25:49,545 p=16573 u=ec2-user n=ansible | Parsed /home/ec2-user/KUBERNETES_TERRAFORM_ANSIBLE/hosts inventory source with ini plugin
2022-06-12 16:25:49,646 p=16573 u=ec2-user n=ansible | redirecting (type: modules) ansible.builtin.k8s to kubernetes.core.k8s
2022-06-12 16:25:49,650 p=16573 u=ec2-user n=ansible | redirecting (type: modules) ansible.builtin.k8s to kubernetes.core.k8s
2022-06-12 16:25:49,651 p=16573 u=ec2-user n=ansible | redirecting (type: modules) ansible.builtin.k8s to kubernetes.core.k8s
2022-06-12 16:25:49,652 p=16573 u=ec2-user n=ansible | redirecting (type: modules) ansible.builtin.k8s to kubernetes.core.k8s
2022-06-12 16:25:49,661 p=16573 u=ec2-user n=ansible | Skipping callback 'default', as we already have a stdout callback.
2022-06-12 16:25:49,661 p=16573 u=ec2-user n=ansible | Skipping callback 'minimal', as we already have a stdout callback.
2022-06-12 16:25:49,662 p=16573 u=ec2-user n=ansible | Skipping callback 'oneline', as we already have a stdout callback.
2022-06-12 16:25:49,662 p=16573 u=ec2-user n=ansible | PLAYBOOK: deployment1.yaml **************************************************************************************************************
2022-06-12 16:25:49,662 p=16573 u=ec2-user n=ansible | 1 plays in deployment1.yaml
2022-06-12 16:25:49,665 p=16573 u=ec2-user n=ansible | PLAY [Ansible k8s module example] *******************************************************************************************************
2022-06-12 16:25:49,680 p=16573 u=ec2-user n=ansible | TASK [Gathering Facts] ******************************************************************************************************************
2022-06-12 16:25:49,708 p=16579 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:49,709 p=16579 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'echo ~ubuntu && sleep 0'"'"''
2022-06-12 16:25:50,222 p=16579 u=ec2-user n=ansible | <3.233.240.89> (0, b'/home/ubuntu\n', b'')
2022-06-12 16:25:50,223 p=16579 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:50,224 p=16579 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'( umask 77 && mkdir -p "` echo /home/ubuntu/.ansible/tmp `"&& mkdir "` echo /home/ubuntu/.ansible/tmp/ansible-tmp-1655051150.2229626-16579-206085591693307 `" && echo ansible-tmp-1655051150.2229626-16579-206085591693307="` echo /home/ubuntu/.ansible/tmp/ansible-tmp-1655051150.2229626-16579-206085591693307 `" ) && sleep 0'"'"''
2022-06-12 16:25:50,240 p=16579 u=ec2-user n=ansible | <3.233.240.89> (0, b'ansible-tmp-1655051150.2229626-16579-206085591693307=/home/ubuntu/.ansible/tmp/ansible-tmp-1655051150.2229626-16579-206085591693307\n', b'')
2022-06-12 16:25:50,537 p=16579 u=ec2-user n=ansible | <3.233.240.89> Attempting python interpreter discovery
2022-06-12 16:25:50,538 p=16579 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:50,539 p=16579 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'echo PLATFORM; uname; echo FOUND; command -v '"'"'"'"'"'"'"'"'/usr/bin/python'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.9'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.8'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.7'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.6'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python3.5'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python2.7'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python2.6'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'/usr/libexec/platform-python'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'/usr/bin/python3'"'"'"'"'"'"'"'"'; command -v '"'"'"'"'"'"'"'"'python'"'"'"'"'"'"'"'"'; echo ENDFOUND && sleep 0'"'"''
2022-06-12 16:25:50,554 p=16579 u=ec2-user n=ansible | <3.233.240.89> (0, b'PLATFORM\nLinux\nFOUND\n/usr/bin/python3.8\n/usr/bin/python3\nENDFOUND\n', b'')
2022-06-12 16:25:50,555 p=16579 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:50,556 p=16579 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'/usr/bin/python3.8 && sleep 0'"'"''
2022-06-12 16:25:50,600 p=16579 u=ec2-user n=ansible | <3.233.240.89> (0, b'{"platform_dist_result": [], "osrelease_content": "NAME=\\"Ubuntu\\"\\nVERSION=\\"20.04.4 LTS (Focal Fossa)\\"\\nID=ubuntu\\nID_LIKE=debian\\nPRETTY_NAME=\\"Ubuntu 20.04.4 LTS\\"\\nVERSION_ID=\\"20.04\\"\\nHOME_URL=\\"https://www.ubuntu.com/\\"\\nSUPPORT_URL=\\"https://help.ubuntu.com/\\"\\nBUG_REPORT_URL=\\"https://bugs.launchpad.net/ubuntu/\\"\\nPRIVACY_POLICY_URL=\\"https://www.ubuntu.com/legal/terms-and-policies/privacy-policy\\"\\nVERSION_CODENAME=focal\\nUBUNTU_CODENAME=focal\\n"}\n', b'')
2022-06-12 16:25:50,602 p=16579 u=ec2-user n=ansible | Using module file /home/ec2-user/.local/lib/python3.7/site-packages/ansible/modules/setup.py
2022-06-12 16:25:50,603 p=16579 u=ec2-user n=ansible | <3.233.240.89> PUT /home/ec2-user/.ansible/tmp/ansible-local-16573wp73zjup/tmpqtmkj51p TO /home/ubuntu/.ansible/tmp/ansible-tmp-1655051150.2229626-16579-206085591693307/AnsiballZ_setup.py
2022-06-12 16:25:50,604 p=16579 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC sftp -b - -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' '[3.233.240.89]'
2022-06-12 16:25:50,647 p=16579 u=ec2-user n=ansible | <3.233.240.89> (0, b'sftp> put /home/ec2-user/.ansible/tmp/ansible-local-16573wp73zjup/tmpqtmkj51p /home/ubuntu/.ansible/tmp/ansible-tmp-1655051150.2229626-16579-206085591693307/AnsiballZ_setup.py\n', b'')
2022-06-12 16:25:50,648 p=16579 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:50,648 p=16579 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'chmod u+x /home/ubuntu/.ansible/tmp/ansible-tmp-1655051150.2229626-16579-206085591693307/ /home/ubuntu/.ansible/tmp/ansible-tmp-1655051150.2229626-16579-206085591693307/AnsiballZ_setup.py && sleep 0'"'"''
2022-06-12 16:25:50,663 p=16579 u=ec2-user n=ansible | <3.233.240.89> (0, b'', b'')
2022-06-12 16:25:50,664 p=16579 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:50,664 p=16579 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' -tt 3.233.240.89 '/bin/sh -c '"'"'sudo -H -S -n  -u root /bin/sh -c '"'"'"'"'"'"'"'"'echo BECOME-SUCCESS-rasaoiumhwxvooagdkzyqqdkjxlhehkc ; /usr/bin/python3 /home/ubuntu/.ansible/tmp/ansible-tmp-1655051150.2229626-16579-206085591693307/AnsiballZ_setup.py'"'"'"'"'"'"'"'"' && sleep 0'"'"''
2022-06-12 16:25:50,687 p=16579 u=ec2-user n=ansible | Escalation succeeded
2022-06-12 16:25:56,223 p=16579 u=ec2-user n=ansible | <3.233.240.89> (0, b'\r\n{"ansible_facts": {"ansible_apparmor": {"status": "enabled"}, "ansible_system": "Linux", "ansible_kernel": "5.13.0-1022-aws", "ansible_kernel_version": "#24~20.04.1-Ubuntu SMP Thu Apr 7 22:10:15 UTC 2022", "ansible_machine": "x86_64", "ansible_python_version": "3.8.10", "ansible_fqdn": "ip-10-0-3-49.ec2.internal", "ansible_hostname": "ip-10-0-3-49", "ansible_nodename": "ip-10-0-3-49", "ansible_domain": "ec2.internal", "ansible_userspace_bits": "64", "ansible_architecture": "x86_64", "ansible_userspace_architecture": "x86_64", "ansible_machine_id": "db9286c05bc54079848ee470ed78801c", "ansible_distribution": "Ubuntu", "ansible_distribution_release": "focal", "ansible_distribution_version": "20.04", "ansible_distribution_major_version": "20", "ansible_distribution_file_path": "/etc/os-release", "ansible_distribution_file_variety": "Debian", "ansible_distribution_file_parsed": true, "ansible_os_family": "Debian", "ansible_user_id": "root", "ansible_user_uid": 0, "ansible_user_gid": 0, "ansible_user_gecos": "root", "ansible_user_dir": "/root", "ansible_user_shell": "/bin/bash", "ansible_real_user_id": 0, "ansible_effective_user_id": 0, "ansible_real_group_id": 0, "ansible_effective_group_id": 0, "ansible_virtualization_type": "xen", "ansible_virtualization_role": "guest", "ansible_virtualization_tech_guest": ["xen"], "ansible_virtualization_tech_host": [], "ansible_system_capabilities_enforced": "False", "ansible_system_capabilities": [], "ansible_is_chroot": false, "ansible_ssh_host_key_dsa_public": "AAAAB3NzaC1kc3MAAACBAJKEuAQFJsmCCHpZk4yVSNAHmVOHGdTTtvSfI4YXt27KcFvEp0eLKC8Toiw7q0JsMQCWuKCNGnc9vbGMpm5lkeqNBWBxyYisGjfejeaG3nF2QtV/S/M1B+DQE1JTSR09C6dwbe1bUCr5tUSHLN64SV8a05d3iZ82wJZQNRObEFcDAAAAFQCzbb7T4lI84Yjqd1IVaGx/qvJ+QwAAAIAt9IWUHidnByfNEd/lLhSjsV5sBHXvL4O41uvK4QVN2Qk7fbpqn6fPahthrcK6RkON6PRHcmx9+G+J/FTzifH9Ie05EcPHp6nmwVodVxmBbJbbhT8w5G8NqnSmAU5iCHgfjiFaT7T2ll3XHv0LmEFnrqjBtWsLocUmeKuanNDrBAAAAIAvoPWfa9/QW+zJCaP2oNnVgIFlqMQHepbq/hC5jTU1pD++eIQT7xDcMFZYaTzckbDoDkw2fGT4K5xV8Md7dvD1QuHdixjBUZ2Jiqduz3nX7YCQ9C2047zME5ZQL+ovH46/iBFY2/vLuCTZpAYhdJXszQPDtScDfVm97mpUQI2Rtg==", "ansible_ssh_host_key_dsa_public_keytype": "ssh-dss", "ansible_ssh_host_key_rsa_public": "AAAAB3NzaC1yc2EAAAADAQABAAABgQDAF1cVN5xWMc/Ab/W6R2CRe29YZDq2lNgPqvv76wm69owMd9Iy9IV6jS6Yu1RIcg+3ESx2ayb5q3C703c6Mn5qruFI0k6Tep/tub2NFp6iL0SkxfA6jGLmOf+nKPZuB8Ipd6SHBNS1tK0EMGYTaT9Rw7JHkbcZX4noxK8/cs1VBQZJ7A/V13gUQFOSLSrIB3yVszk4o7HdXlpgphPpL8S97X2gY5w2p+NGwMeLxgHuA2d3Hmd06aQ3Wl4tzjBTAiLe50SguFkfjzceSa+BcpVz2oywIfaImatKQ8TaMjviwT5yn0rdgF6K1CeK4KmXnpZDR67PxVXWd+KdcEk9xXBzVkUbRyxSrygQ5MwTWtWDMB3vm30w+hZPL/BJMwWYv+0Fje71aSmuumdHwVjh/5KnufmSfd/77olprjFjLz/Lo0/ig8xFJjHo3AEZRzgfg/6s0G+h2T4c3Au2Prtu+GxoggIv8t8z060hOHJz5CoM+Df5zf7CJzeLjcD+gaeXmpk=", "ansible_ssh_host_key_rsa_public_keytype": "ssh-rsa", "ansible_ssh_host_key_ecdsa_public": "AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBJeCp1QZ2xek4hEIk8f4nm9oLnxM7qUiZ8vm2LQHQGlDUCh14qYNsYp6k9vJCHJucRgEWjyvCEEY9HDzfUActVM=", "ansible_ssh_host_key_ecdsa_public_keytype": "ecdsa-sha2-nistp256", "ansible_ssh_host_key_ed25519_public": "AAAAC3NzaC1lZDI1NTE5AAAAIOxkHjArjHGBuIwOMOsPxyRtS4AR8g2O6Oksxj1aly/f", "ansible_ssh_host_key_ed25519_public_keytype": "ssh-ed25519", "ansible_date_time": {"year": "2022", "month": "06", "weekday": "Sunday", "weekday_number": "0", "weeknumber": "23", "day": "12", "hour": "16", "minute": "25", "second": "51", "epoch": "1655051151", "date": "2022-06-12", "time": "16:25:51", "iso8601_micro": "2022-06-12T16:25:51.018821Z", "iso8601": "2022-06-12T16:25:51Z", "iso8601_basic": "20220612T162551018821", "iso8601_basic_short": "20220612T162551", "tz": "UTC", "tz_dst": "UTC", "tz_offset": "+0000"}, "ansible_fips": false, "ansible_iscsi_iqn": "", "ansible_processor": ["0", "GenuineIntel", "Intel(R) Xeon(R) CPU E5-2676 v3 @ 2.40GHz"], "ansible_processor_count": 1, "ansible_processor_cores": 1, "ansible_processor_threads_per_core": 1, "ansible_processor_vcpus": 1, "ansible_processor_nproc": 1, "ansible_memtotal_mb": 967, "ansible_memfree_mb": 136, "ansible_swaptotal_mb": 0, "ansible_swapfree_mb": 0, "ansible_memory_mb": {"real": {"total": 967, "used": 831, "free": 136}, "nocache": {"free": 680, "used": 287}, "swap": {"total": 0, "free": 0, "used": 0, "cached": 0}}, "ansible_bios_date": "08/24/2006", "ansible_bios_vendor": "Xen", "ansible_bios_version": "4.11.amazon", "ansible_board_asset_tag": "NA", "ansible_board_name": "NA", "ansible_board_serial": "NA", "ansible_board_vendor": "NA", "ansible_board_version": "NA", "ansible_chassis_asset_tag": "NA", "ansible_chassis_serial": "NA", "ansible_chassis_vendor": "Xen", "ansible_chassis_version": "NA", "ansible_form_factor": "Other", "ansible_product_name": "HVM domU", "ansible_product_serial": "ec2affd0-59d5-94e4-02bf-5bb06435f434", "ansible_product_uuid": "ec2affd0-59d5-94e4-02bf-5bb06435f434", "ansible_product_version": "4.11.amazon", "ansible_system_vendor": "Xen", "ansible_devices": {"loop1": {"virtual": 1, "links": {"ids": [], "uuids": [], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "removable": "0", "support_discard": "4096", "partitions": {}, "rotational": "0", "scheduler_mode": "mq-deadline", "sectors": "113696", "sectorsize": "512", "size": "55.52 MB", "host": "", "holders": []}, "xvda": {"virtual": 1, "links": {"ids": [], "uuids": [], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "removable": "0", "support_discard": "0", "partitions": {"xvda1": {"links": {"ids": [], "uuids": ["048df47e-5bfe-4ce9-8bbc-b65a30843a4e"], "labels": ["cloudimg-rootfs"], "masters": []}, "start": "2048", "sectors": "16775135", "sectorsize": 512, "size": "8.00 GB", "uuid": "048df47e-5bfe-4ce9-8bbc-b65a30843a4e", "holders": []}}, "rotational": "0", "scheduler_mode": "mq-deadline", "sectors": "16777216", "sectorsize": "512", "size": "8.00 GB", "host": "", "holders": []}, "loop8": {"virtual": 1, "links": {"ids": [], "uuids": [], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "removable": "0", "support_discard": "4096", "partitions": {}, "rotational": "0", "scheduler_mode": "mq-deadline", "sectors": "51416", "sectorsize": "512", "size": "25.11 MB", "host": "", "holders": []}, "loop6": {"virtual": 1, "links": {"ids": [], "uuids": [], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "removable": "0", "support_discard": "4096", "partitions": {}, "rotational": "0", "scheduler_mode": "mq-deadline", "sectors": "113736", "sectorsize": "512", "size": "55.54 MB", "host": "", "holders": []}, "loop4": {"virtual": 1, "links": {"ids": [], "uuids": [], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "removable": "0", "support_discard": "4096", "partitions": {}, "rotational": "0", "scheduler_mode": "mq-deadline", "sectors": "89352", "sectorsize": "512", "size": "43.63 MB", "host": "", "holders": []}, "loop2": {"virtual": 1, "links": {"ids": [], "uuids": [], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "removable": "0", "support_discard": "4096", "partitions": {}, "rotational": "0", "scheduler_mode": "mq-deadline", "sectors": "126760", "sectorsize": "512", "size": "61.89 MB", "host": "", "holders": []}, "loop0": {"virtual": 1, "links": {"ids": [], "uuids": [], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "removable": "0", "support_discard": "4096", "partitions": {}, "rotational": "0", "scheduler_mode": "mq-deadline", "sectors": "54552", "sectorsize": "512", "size": "26.64 MB", "host": "", "holders": []}, "loop7": {"virtual": 1, "links": {"ids": [], "uuids": [], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "removable": "0", "support_discard": "4096", "partitions": {}, "rotational": "0", "scheduler_mode": "mq-deadline", "sectors": "126824", "sectorsize": "512", "size": "61.93 MB", "host": "", "holders": []}, "loop5": {"virtual": 1, "links": {"ids": [], "uuids": [], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "removable": "0", "support_discard": "4096", "partitions": {}, "rotational": "0", "scheduler_mode": "mq-deadline", "sectors": "96160", "sectorsize": "512", "size": "46.95 MB", "host": "", "holders": []}, "loop3": {"virtual": 1, "links": {"ids": [], "uuids": [], "labels": [], "masters": []}, "vendor": null, "model": null, "sas_address": null, "sas_device_handle": null, "removable": "0", "support_discard": "4096", "partitions": {}, "rotational": "0", "scheduler_mode": "mq-deadline", "sectors": "138880", "sectorsize": "512", "size": "67.81 MB", "host": "", "holders": []}}, "ansible_device_links": {"ids": {}, "uuids": {"xvda1": ["048df47e-5bfe-4ce9-8bbc-b65a30843a4e"]}, "labels": {"xvda1": ["cloudimg-rootfs"]}, "masters": {}}, "ansible_uptime_seconds": 100140, "ansible_lvm": {"lvs": {}, "vgs": {}, "pvs": {}}, "ansible_mounts": [{"mount": "/", "device": "/dev/root", "fstype": "ext4", "options": "rw,relatime,discard", "size_total": 8259014656, "size_available": 5748658176, "block_size": 4096, "block_total": 2016361, "block_available": 1403481, "block_used": 612880, "inode_total": 1024000, "inode_available": 913801, "inode_used": 110199, "uuid": "048df47e-5bfe-4ce9-8bbc-b65a30843a4e"}, {"mount": "/snap/lxd/22753", "device": "/dev/loop3", "fstype": "squashfs", "options": "ro,nodev,relatime", "size_total": 71172096, "size_available": 0, "block_size": 131072, "block_total": 543, "block_available": 0, "block_used": 543, "inode_total": 802, "inode_available": 0, "inode_used": 802, "uuid": "N/A"}, {"mount": "/snap/core20/1405", "device": "/dev/loop2", "fstype": "squashfs", "options": "ro,nodev,relatime", "size_total": 65011712, "size_available": 0, "block_size": 131072, "block_total": 496, "block_available": 0, "block_used": 496, "inode_total": 11778, "inode_available": 0, "inode_used": 11778, "uuid": "N/A"}, {"mount": "/snap/core18/2344", "device": "/dev/loop1", "fstype": "squashfs", "options": "ro,nodev,relatime", "size_total": 58327040, "size_available": 0, "block_size": 131072, "block_total": 445, "block_available": 0, "block_used": 445, "inode_total": 10849, "inode_available": 0, "inode_used": 10849, "uuid": "N/A"}, {"mount": "/snap/amazon-ssm-agent/5163", "device": "/dev/loop0", "fstype": "squashfs", "options": "ro,nodev,relatime", "size_total": 28049408, "size_available": 0, "block_size": 131072, "block_total": 214, "block_available": 0, "block_used": 214, "inode_total": 16, "inode_available": 0, "inode_used": 16, "uuid": "N/A"}, {"mount": "/snap/snapd/15177", "device": "/dev/loop4", "fstype": "squashfs", "options": "ro,nodev,relatime", "size_total": 45875200, "size_available": 0, "block_size": 131072, "block_total": 350, "block_available": 0, "block_used": 350, "inode_total": 482, "inode_available": 0, "inode_used": 482, "uuid": "N/A"}, {"mount": "/snap/snapd/16010", "device": "/dev/loop5", "fstype": "squashfs", "options": "ro,nodev,relatime", "size_total": 49283072, "size_available": 0, "block_size": 131072, "block_total": 376, "block_available": 0, "block_used": 376, "inode_total": 486, "inode_available": 0, "inode_used": 486, "uuid": "N/A"}, {"mount": "/snap/core18/2409", "device": "/dev/loop6", "fstype": "squashfs", "options": "ro,nodev,relatime", "size_total": 58327040, "size_available": 0, "block_size": 131072, "block_total": 445, "block_available": 0, "block_used": 445, "inode_total": 10857, "inode_available": 0, "inode_used": 10857, "uuid": "N/A"}, {"mount": "/snap/core20/1518", "device": "/dev/loop7", "fstype": "squashfs", "options": "ro,nodev,relatime", "size_total": 65011712, "size_available": 0, "block_size": 131072, "block_total": 496, "block_available": 0, "block_used": 496, "inode_total": 11789, "inode_available": 0, "inode_used": 11789, "uuid": "N/A"}, {"mount": "/snap/amazon-ssm-agent/5656", "device": "/dev/loop8", "fstype": "squashfs", "options": "ro,nodev,relatime", "size_total": 26345472, "size_available": 0, "block_size": 131072, "block_total": 201, "block_available": 0, "block_used": 201, "inode_total": 16, "inode_available": 0, "inode_used": 16, "uuid": "N/A"}], "ansible_hostnqn": "", "ansible_local": {}, "ansible_python": {"version": {"major": 3, "minor": 8, "micro": 10, "releaselevel": "final", "serial": 0}, "version_info": [3, 8, 10, "final", 0], "executable": "/usr/bin/python3", "has_sslcontext": true, "type": "cpython"}, "ansible_env": {"SUDO_GID": "1000", "MAIL": "/var/mail/root", "USER": "root", "HOME": "/root", "SUDO_UID": "1000", "LOGNAME": "root", "TERM": "xterm", "PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin", "LANG": "C.UTF-8", "SUDO_COMMAND": "/bin/sh -c echo BECOME-SUCCESS-rasaoiumhwxvooagdkzyqqdkjxlhehkc ; /usr/bin/python3 /home/ubuntu/.ansible/tmp/ansible-tmp-1655051150.2229626-16579-206085591693307/AnsiballZ_setup.py", "SHELL": "/bin/bash", "SUDO_USER": "ubuntu", "PWD": "/home/ubuntu"}, "ansible_cmdline": {"BOOT_IMAGE": "/boot/vmlinuz-5.13.0-1022-aws", "root": "PARTUUID=b236b0c8-01", "ro": true, "console": "ttyS0", "nvme_core.io_timeout": "4294967295", "panic": "-1"}, "ansible_proc_cmdline": {"BOOT_IMAGE": "/boot/vmlinuz-5.13.0-1022-aws", "root": "PARTUUID=b236b0c8-01", "ro": true, "console": ["tty1", "ttyS0"], "nvme_core.io_timeout": "4294967295", "panic": "-1"}, "ansible_interfaces": ["lo", "eth0"], "ansible_eth0": {"device": "eth0", "macaddress": "02:60:16:ef:54:85", "mtu": 9001, "active": true, "module": "xen_netfront", "type": "ether", "pciid": "vif-0", "promisc": false, "ipv4": {"address": "10.0.3.49", "broadcast": "10.0.3.255", "netmask": "255.255.255.0", "network": "10.0.3.0"}, "ipv6": [{"address": "fe80::60:16ff:feef:5485", "prefix": "64", "scope": "link"}], "features": {"rx_checksumming": "on [fixed]", "tx_checksumming": "on", "tx_checksum_ipv4": "on [fixed]", "tx_checksum_ip_generic": "off [fixed]", "tx_checksum_ipv6": "on", "tx_checksum_fcoe_crc": "off [fixed]", "tx_checksum_sctp": "off [fixed]", "scatter_gather": "on", "tx_scatter_gather": "on", "tx_scatter_gather_fraglist": "off [fixed]", "tcp_segmentation_offload": "on", "tx_tcp_segmentation": "on", "tx_tcp_ecn_segmentation": "off [fixed]", "tx_tcp_mangleid_segmentation": "off", "tx_tcp6_segmentation": "on", "generic_segmentation_offload": "on", "generic_receive_offload": "on", "large_receive_offload": "off [fixed]", "rx_vlan_offload": "off [fixed]", "tx_vlan_offload": "off [fixed]", "ntuple_filters": "off [fixed]", "receive_hashing": "off [fixed]", "highdma": "off [fixed]", "rx_vlan_filter": "off [fixed]", "vlan_challenged": "off [fixed]", "tx_lockless": "off [fixed]", "netns_local": "off [fixed]", "tx_gso_robust": "on [fixed]", "tx_fcoe_segmentation": "off [fixed]", "tx_gre_segmentation": "off [fixed]", "tx_gre_csum_segmentation": "off [fixed]", "tx_ipxip4_segmentation": "off [fixed]", "tx_ipxip6_segmentation": "off [fixed]", "tx_udp_tnl_segmentation": "off [fixed]", "tx_udp_tnl_csum_segmentation": "off [fixed]", "tx_gso_partial": "off [fixed]", "tx_tunnel_remcsum_segmentation": "off [fixed]", "tx_sctp_segmentation": "off [fixed]", "tx_esp_segmentation": "off [fixed]", "tx_udp_segmentation": "off [fixed]", "tx_gso_list": "off [fixed]", "fcoe_mtu": "off [fixed]", "tx_nocache_copy": "off", "loopback": "off [fixed]", "rx_fcs": "off [fixed]", "rx_all": "off [fixed]", "tx_vlan_stag_hw_insert": "off [fixed]", "rx_vlan_stag_hw_parse": "off [fixed]", "rx_vlan_stag_filter": "off [fixed]", "l2_fwd_offload": "off [fixed]", "hw_tc_offload": "off [fixed]", "esp_hw_offload": "off [fixed]", "esp_tx_csum_hw_offload": "off [fixed]", "rx_udp_tunnel_port_offload": "off [fixed]", "tls_hw_tx_offload": "off [fixed]", "tls_hw_rx_offload": "off [fixed]", "rx_gro_hw": "off [fixed]", "tls_hw_record": "off [fixed]", "rx_gro_list": "off", "macsec_hw_offload": "off [fixed]", "rx_udp_gro_forwarding": "off", "hsr_tag_ins_offload": "off [fixed]", "hsr_tag_rm_offload": "off [fixed]", "hsr_fwd_offload": "off [fixed]", "hsr_dup_offload": "off [fixed]"}, "timestamping": ["tx_software", "rx_software", "software"], "hw_timestamp_filters": []}, "ansible_lo": {"device": "lo", "mtu": 65536, "active": true, "type": "loopback", "promisc": false, "ipv4": {"address": "127.0.0.1", "broadcast": "", "netmask": "255.0.0.0", "network": "127.0.0.0"}, "ipv6": [{"address": "::1", "prefix": "128", "scope": "host"}], "features": {"rx_checksumming": "on [fixed]", "tx_checksumming": "on", "tx_checksum_ipv4": "off [fixed]", "tx_checksum_ip_generic": "on [fixed]", "tx_checksum_ipv6": "off [fixed]", "tx_checksum_fcoe_crc": "off [fixed]", "tx_checksum_sctp": "on [fixed]", "scatter_gather": "on", "tx_scatter_gather": "on [fixed]", "tx_scatter_gather_fraglist": "on [fixed]", "tcp_segmentation_offload": "on", "tx_tcp_segmentation": "on", "tx_tcp_ecn_segmentation": "on", "tx_tcp_mangleid_segmentation": "on", "tx_tcp6_segmentation": "on", "generic_segmentation_offload": "on", "generic_receive_offload": "on", "large_receive_offload": "off [fixed]", "rx_vlan_offload": "off [fixed]", "tx_vlan_offload": "off [fixed]", "ntuple_filters": "off [fixed]", "receive_hashing": "off [fixed]", "highdma": "on [fixed]", "rx_vlan_filter": "off [fixed]", "vlan_challenged": "on [fixed]", "tx_lockless": "on [fixed]", "netns_local": "on [fixed]", "tx_gso_robust": "off [fixed]", "tx_fcoe_segmentation": "off [fixed]", "tx_gre_segmentation": "off [fixed]", "tx_gre_csum_segmentation": "off [fixed]", "tx_ipxip4_segmentation": "off [fixed]", "tx_ipxip6_segmentation": "off [fixed]", "tx_udp_tnl_segmentation": "off [fixed]", "tx_udp_tnl_csum_segmentation": "off [fixed]", "tx_gso_partial": "off [fixed]", "tx_tunnel_remcsum_segmentation": "off [fixed]", "tx_sctp_segmentation": "on", "tx_esp_segmentation": "off [fixed]", "tx_udp_segmentation": "on", "tx_gso_list": "on", "fcoe_mtu": "off [fixed]", "tx_nocache_copy": "off [fixed]", "loopback": "on [fixed]", "rx_fcs": "off [fixed]", "rx_all": "off [fixed]", "tx_vlan_stag_hw_insert": "off [fixed]", "rx_vlan_stag_hw_parse": "off [fixed]", "rx_vlan_stag_filter": "off [fixed]", "l2_fwd_offload": "off [fixed]", "hw_tc_offload": "off [fixed]", "esp_hw_offload": "off [fixed]", "esp_tx_csum_hw_offload": "off [fixed]", "rx_udp_tunnel_port_offload": "off [fixed]", "tls_hw_tx_offload": "off [fixed]", "tls_hw_rx_offload": "off [fixed]", "rx_gro_hw": "off [fixed]", "tls_hw_record": "off [fixed]", "rx_gro_list": "off", "macsec_hw_offload": "off [fixed]", "rx_udp_gro_forwarding": "off", "hsr_tag_ins_offload": "off [fixed]", "hsr_tag_rm_offload": "off [fixed]", "hsr_fwd_offload": "off [fixed]", "hsr_dup_offload": "off [fixed]"}, "timestamping": ["tx_software", "rx_software", "software"], "hw_timestamp_filters": []}, "ansible_default_ipv4": {"gateway": "10.0.3.1", "interface": "eth0", "address": "10.0.3.49", "broadcast": "10.0.3.255", "netmask": "255.255.255.0", "network": "10.0.3.0", "macaddress": "02:60:16:ef:54:85", "mtu": 9001, "type": "ether", "alias": "eth0"}, "ansible_default_ipv6": {}, "ansible_all_ipv4_addresses": ["10.0.3.49"], "ansible_all_ipv6_addresses": ["fe80::60:16ff:feef:5485"], "ansible_selinux_python_present": true, "ansible_selinux": {"status": "disabled"}, "ansible_lsb": {"id": "Ubuntu", "description": "Ubuntu 20.04.4 LTS", "release": "20.04", "codename": "focal", "major_release": "20"}, "ansible_fibre_channel_wwn": [], "ansible_service_mgr": "systemd", "ansible_dns": {"nameservers": ["127.0.0.53"], "options": {"edns0": true, "trust-ad": true}, "search": ["ec2.internal"]}, "ansible_pkg_mgr": "apt", "gather_subset": ["all"], "module_setup": true}, "invocation": {"module_args": {"gather_subset": ["all"], "gather_timeout": 10, "filter": [], "fact_path": "/etc/ansible/facts.d"}}}\r\n', b'Shared connection to 3.233.240.89 closed.\r\n')
2022-06-12 16:25:56,228 p=16579 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:56,229 p=16579 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'rm -f -r /home/ubuntu/.ansible/tmp/ansible-tmp-1655051150.2229626-16579-206085591693307/ > /dev/null 2>&1 && sleep 0'"'"''
2022-06-12 16:25:56,245 p=16579 u=ec2-user n=ansible | <3.233.240.89> (0, b'', b'')
2022-06-12 16:25:56,263 p=16573 u=ec2-user n=ansible | ok: [3.233.240.89]
2022-06-12 16:25:56,280 p=16573 u=ec2-user n=ansible | META: ran handlers
2022-06-12 16:25:56,295 p=16573 u=ec2-user n=ansible | TASK [Create a namespaces] **************************************************************************************************************
2022-06-12 16:25:56,323 p=16593 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:56,324 p=16593 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'echo ~ubuntu && sleep 0'"'"''
2022-06-12 16:25:56,339 p=16593 u=ec2-user n=ansible | <3.233.240.89> (0, b'/home/ubuntu\n', b'')
2022-06-12 16:25:56,340 p=16593 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:56,340 p=16593 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'( umask 77 && mkdir -p "` echo /home/ubuntu/.ansible/tmp `"&& mkdir "` echo /home/ubuntu/.ansible/tmp/ansible-tmp-1655051156.3399289-16593-157872983204001 `" && echo ansible-tmp-1655051156.3399289-16593-157872983204001="` echo /home/ubuntu/.ansible/tmp/ansible-tmp-1655051156.3399289-16593-157872983204001 `" ) && sleep 0'"'"''
2022-06-12 16:25:56,356 p=16593 u=ec2-user n=ansible | <3.233.240.89> (0, b'ansible-tmp-1655051156.3399289-16593-157872983204001=/home/ubuntu/.ansible/tmp/ansible-tmp-1655051156.3399289-16593-157872983204001\n', b'')
2022-06-12 16:25:56,357 p=16593 u=ec2-user n=ansible | redirecting (type: modules) ansible.builtin.k8s to kubernetes.core.k8s
2022-06-12 16:25:56,524 p=16593 u=ec2-user n=ansible | Using module file /home/ec2-user/.local/lib/python3.7/site-packages/ansible_collections/kubernetes/core/plugins/modules/k8s.py
2022-06-12 16:25:56,525 p=16593 u=ec2-user n=ansible | <3.233.240.89> PUT /home/ec2-user/.ansible/tmp/ansible-local-16573wp73zjup/tmpzgbj59bs TO /home/ubuntu/.ansible/tmp/ansible-tmp-1655051156.3399289-16593-157872983204001/AnsiballZ_k8s.py
2022-06-12 16:25:56,526 p=16593 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC sftp -b - -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' '[3.233.240.89]'
2022-06-12 16:25:56,560 p=16593 u=ec2-user n=ansible | <3.233.240.89> (0, b'sftp> put /home/ec2-user/.ansible/tmp/ansible-local-16573wp73zjup/tmpzgbj59bs /home/ubuntu/.ansible/tmp/ansible-tmp-1655051156.3399289-16593-157872983204001/AnsiballZ_k8s.py\n', b'')
2022-06-12 16:25:56,560 p=16593 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:56,561 p=16593 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'chmod u+x /home/ubuntu/.ansible/tmp/ansible-tmp-1655051156.3399289-16593-157872983204001/ /home/ubuntu/.ansible/tmp/ansible-tmp-1655051156.3399289-16593-157872983204001/AnsiballZ_k8s.py && sleep 0'"'"''
2022-06-12 16:25:56,575 p=16593 u=ec2-user n=ansible | <3.233.240.89> (0, b'', b'')
2022-06-12 16:25:56,576 p=16593 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:56,577 p=16593 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' -tt 3.233.240.89 '/bin/sh -c '"'"'sudo -H -S -n  -u root /bin/sh -c '"'"'"'"'"'"'"'"'echo BECOME-SUCCESS-zchtzimpbyvfrvxzpjukfjpkeryapmsn ; /usr/bin/python3 /home/ubuntu/.ansible/tmp/ansible-tmp-1655051156.3399289-16593-157872983204001/AnsiballZ_k8s.py'"'"'"'"'"'"'"'"' && sleep 0'"'"''
2022-06-12 16:25:56,596 p=16593 u=ec2-user n=ansible | Escalation succeeded
2022-06-12 16:25:56,823 p=16593 u=ec2-user n=ansible | <3.233.240.89> (1, b'\r\n{"exception": "Traceback (most recent call last):\\n  File \\"/tmp/ansible_k8s_payload_a_d47uwp/ansible_k8s_payload.zip/ansible_collections/kubernetes/core/plugins/module_utils/common.py\\", line 90, in <module>\\n    from openshift.dynamic.resource import ResourceInstance\\nModuleNotFoundError: No module named \'openshift\'\\n\\nDuring handling of the above exception, another exception occurred:\\n\\nTraceback (most recent call last):\\n  File \\"/tmp/ansible_k8s_payload_a_d47uwp/ansible_k8s_payload.zip/ansible_collections/kubernetes/core/plugins/module_utils/common.py\\", line 93, in <module>\\n    from openshift.dynamic.client import ResourceInstance\\nModuleNotFoundError: No module named \'openshift\'\\n", "error": "No module named \'openshift\'", "failed": true, "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49\'s Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter", "invocation": {"module_args": {"name": "cloudmotos", "api_version": "v1", "kind": "Namespace", "state": "present", "force": false, "wait": false, "wait_sleep": 5, "wait_timeout": 120, "append_hash": false, "apply": false, "namespace": null, "resource_definition": null, "src": null, "kubeconfig": null, "context": null, "host": null, "api_key": null, "username": null, "password": null, "validate_certs": null, "ca_cert": null, "client_cert": null, "client_key": null, "proxy": null, "persist_config": null, "wait_condition": null, "merge_type": null, "validate": null, "template": null, "delete_options": null}}}\r\n', b'Shared connection to 3.233.240.89 closed.\r\n')
2022-06-12 16:25:56,823 p=16593 u=ec2-user n=ansible | <3.233.240.89> Failed to connect to the host via ssh: Shared connection to 3.233.240.89 closed.
2022-06-12 16:25:56,824 p=16593 u=ec2-user n=ansible | <3.233.240.89> ESTABLISH SSH CONNECTION FOR USER: ubuntu
2022-06-12 16:25:56,825 p=16593 u=ec2-user n=ansible | <3.233.240.89> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o 'IdentityFile="/home/ec2-user/eks-cluster.pem"' -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="ubuntu"' -o ConnectTimeout=10 -o 'ControlPath="/home/ec2-user/.ansible/cp/9e1608655e"' 3.233.240.89 '/bin/sh -c '"'"'rm -f -r /home/ubuntu/.ansible/tmp/ansible-tmp-1655051156.3399289-16593-157872983204001/ > /dev/null 2>&1 && sleep 0'"'"''
2022-06-12 16:25:56,840 p=16593 u=ec2-user n=ansible | <3.233.240.89> (0, b'', b'')
2022-06-12 16:25:56,843 p=16573 u=ec2-user n=ansible | The full traceback is:
Traceback (most recent call last):
  File "/tmp/ansible_k8s_payload_a_d47uwp/ansible_k8s_payload.zip/ansible_collections/kubernetes/core/plugins/module_utils/common.py", line 90, in <module>
    from openshift.dynamic.resource import ResourceInstance
ModuleNotFoundError: No module named 'openshift'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/ansible_k8s_payload_a_d47uwp/ansible_k8s_payload.zip/ansible_collections/kubernetes/core/plugins/module_utils/common.py", line 93, in <module>
    from openshift.dynamic.client import ResourceInstance
ModuleNotFoundError: No module named 'openshift'

2022-06-12 16:25:56,844 p=16573 u=ec2-user n=ansible | fatal: [3.233.240.89]: FAILED! => {
    "changed": false,
    "error": "No module named 'openshift'",
    "invocation": {
        "module_args": {
            "api_key": null,
            "api_version": "v1",
            "append_hash": false,
            "apply": false,
            "ca_cert": null,
            "client_cert": null,
            "client_key": null,
            "context": null,
            "delete_options": null,
            "force": false,
            "host": null,
            "kind": "Namespace",
            "kubeconfig": null,
            "merge_type": null,
            "name": "cloudmotos",
            "namespace": null,
            "password": null,
            "persist_config": null,
            "proxy": null,
            "resource_definition": null,
            "src": null,
            "state": "present",
            "template": null,
            "username": null,
            "validate": null,
            "validate_certs": null,
            "wait": false,
            "wait_condition": null,
            "wait_sleep": 5,
            "wait_timeout": 120
        }
    },
    "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"
}
2022-06-12 16:25:56,847 p=16573 u=ec2-user n=ansible | PLAY RECAP ******************************************************************************************************************************
2022-06-12 16:25:56,847 p=16573 u=ec2-user n=ansible | 3.233.240.89               : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
2022-06-12 16:26:11,308 p=16604 u=ec2-user n=ansible | PLAY [Ansible k8s module example] *******************************************************************************************************
2022-06-12 16:26:11,324 p=16604 u=ec2-user n=ansible | TASK [Gathering Facts] ******************************************************************************************************************
2022-06-12 16:26:12,568 p=16604 u=ec2-user n=ansible | ok: [3.233.240.89]
2022-06-12 16:26:12,599 p=16604 u=ec2-user n=ansible | TASK [Create a namespaces] **************************************************************************************************************
2022-06-12 16:26:13,148 p=16604 u=ec2-user n=ansible | An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'openshift'
2022-06-12 16:26:13,149 p=16604 u=ec2-user n=ansible | fatal: [3.233.240.89]: FAILED! => {"changed": false, "error": "No module named 'openshift'", "msg": "Failed to import the required Python library (openshift) on ip-10-0-3-49's Python /usr/bin/python3. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter"}
2022-06-12 16:26:13,152 p=16604 u=ec2-user n=ansible | PLAY RECAP ******************************************************************************************************************************
2022-06-12 16:26:13,152 p=16604 u=ec2-user n=ansible | 3.233.240.89               : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   
